src/utils/create-calendars.ts

```diff
@@ -59,6 +59,7 @@ const createCalendar = (
   }
 
   return {
+    $date: offsetDate,
```

Comment:
- what is the use case for this? months has `active` prop for usecase from the PR description.

Reply:
Aha! I didn't find this information before, seems like this does also work:

```ts
const months = useMonths(dpState)
const activeMonth = Object.values(months)[0].find((month) => month.active)
```

This PR might offer some minor convenience then.
@Feshchenko you can decide if you still want that in then :)

Reply:
Yea, let's keep this PR.
You should make this steps:
- rebase with master to get green tests.
- delete `$date` from calendars.
- increase version in the package.json